internal/engine/upper.go

```diff
@@ -15,6 +16,7 @@ func UpService(ctx context.Context, service model.Service, watch bool, stdout io
 	if watch {
 		for {
 			// TODO(matt) actually wait for a file to change instead of building on a loop
+			logger.Get(ctx).Verbose("building and deploying")
```

Comment:
do we expect this to be user-visible?

Reply:
yes, if the user runs with loglevel >= verbose

the message should change (like, at least include a service name), but this PR is more about allowing logging than polishing the UI